includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -591,12 +591,20 @@ static function( $html ) {
 					return $html;
 				}
 
+				$img_width  = $src[1] ? (int) $src[1] : 0;
+				$img_height = $src[2] ? (int) $src[2] : 0;
```

Comment:
According to the docs for this function, the two values will be integers. The failure case is captured by `false` being returned by the function as a whole. Therefore I think this can just do:

```suggestion
				$img_width  = (int) $src[1];
				$img_height = (int) $src[2];
```